README.md

```diff
@@ -8,19 +8,51 @@ Our open source community's Wiki focuses on answering your questions about learn
 
 ## Get a Fork ready
 
-You can get your own fork by using the button or clicking [this](https://github.com/FreeCodeCamp/wiki/new/master?readme=1#fork-destination-box).
-Once you get a fork, you will clone it with `git clone url` where `url` is the link to your fork.
+1. Fork it: You can get your own fork of our wiki by using the button or clicking [this](https://github.com/FreeCodeCamp/wiki/new/master?readme=1#fork-destination-box).
 
-Once that is done, you can start contributing by checking our [issues](https://github.com/FreeCodeCamp/wiki/issues) and creating nee pull requests.
+  ![Fork Button](https://help.github.com/assets/images/help/repository/fork_button.jpg)
 
-## Contribution Guides
+2. Once you have forked the repository, you will clone it with:
+
+  ```sh
+  $git clone https://github.com/YOUR-USERNAME/wiki.git
+  ```
+
+3. Add the remote: You will need a way to keep your fork synchronized with the original repository.
+
+  ```sh
+  $git remote -v
+  origin  https://github.com/YOUR_USERNAME/YOUR_FORK.git (fetch)
+  origin  https://github.com/YOUR_USERNAME/YOUR_FORK.git (push)
+  ```
+
+  This means there is no remote pointing to the original repository you forked from. You would add it with:
+
+  ```sh
+  $git remote add upstream https://github.com/FreeCodeCamp/wiki.git
+  ```
 
-We currently have a couple of guides to help you contribute, via the browser, command line or desktop application.
+  You can verify the changes with `git remote -v`
```

Comment:
> You can **verify** the changes with `git remote -v`

@Rafase282 so that users don't get confused that `-v` flag doesn't mean _verify_.

Or maybe change your wording to

```diff
- You can verify the changes with `git remote -v`
+ You may see the changes using **`git remote -v`**.
```

----
> You may see the changes using **`git remote -v`**.